test/Microsoft.AspNetCore.JsonPatch.Test/Microsoft.AspNetCore.JsonPatch.Test.csproj

```diff
@@ -3,13 +3,21 @@
   <Import Project="..\..\build\common.props" />
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <TargetFrameworks>netcoreapp2.0;net461</TargetFrameworks>
+    <TargetFrameworks Condition=" '$(OS)' != 'Windows_NT' ">netcoreapp2.0</TargetFrameworks>
+    <!-- allow skipped tests -->
```

Comment:
I added this to aspnet/Testing because skipped tests are intentionally part of the testing. In all other repos, we want the warnings so that its easier to identify which tests are being skipped. We should either fixed skipped tests or removed them.